katas/main/src/pokerhands/jorge/Dictionary.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace PokerGame.Dictionary.Jorge;
+
+public class PokerDictionary
```

Comment:
Not sure if it is really necessary to prefix class names with `Poker`, in such case, namespace is used for that